src/components/table/README.md

```diff
@@ -1510,6 +1510,7 @@ functionality is non critical or can be provided via other means:
 - `row-contextmenu`
 - `row-hovered`
 - `row-unhovered`
+- `row-middle-clicked`
```

Comment:
We should probably note that this event is not supported in all browsers (i.e. IE, Safari, and most mobile browsers)